integration-tests/goss/goss-shared.yaml

```diff
@@ -188,11 +188,19 @@ http:
       - "Foo: bar"
     headers: ["Content-Type: application/json"]
     body: ['"Foo": "bar"']
+  https://httpbin.org/headers?host:
```

Comment:
Can you point me to the documentation for `?host`

Reply:
I am just appending an arbitrary GET param that httpbin seems to ignore to prevent duplicating the other check against `https://httpbin.org/headers`. There may be a cleaner way to prevent duplicating the checks, this didn't seem to impact the behavior of httpbin, so I gave it a shot.